themes/bootstrap3/templates/ajax/status-full.phtml

```diff
@@ -12,8 +12,16 @@
         break;
       }
       $callNumPrefix = !empty($item['callnumber_prefix']) ? $item['callnumber_prefix'] . ' ' : '';
+
+      $itemHasAdditionalHoldingsFields = false;
+      foreach ($this->holdingsTextFieldNames as $holdingsTextField) {
+        if ($item[$holdingsTextField]) {
+          $itemHasAdditionalHoldingsFields = true;
+          break;
+        }
+      }
     ?>
-    <tr>
+    <tr<?=$itemHasAdditionalHoldingsFields ? $this->htmlAttributes(['class' => 'itemWithAdditionalHoldingFields']) : ''?>>
```

Comment:
I refactored this slightly to avoid unnecessary extra whitespace in the `<tr>` tag.

I couldn't figure out what it was actually doing, though; it didn't seem to make a difference whether the class was present or not. I was only testing in a bootstrap3-based theme, though, so maybe it's part of the bootstrap5 support you alluded to.

Reply:
Yep, that's right.  I'm trying to collapse the row containing the location/callnumber/status with the rows containing the additional holdings fields.  Without the CSS, it comes out looking like this:

```
-------------------------------------------
Location, callnumber
-------------------------------------------  <<< extra divider
  Holding note 1
  Holding note 2
-------------------------------------------
```

But I didn't want that extra divider between the rows because I felt like it obscured the relationship between the holdings and the notes.

In bootstrap3, that just required styling the new holdings fields `tr` elements to remove border-top.  But in bootstrap5, I had to style the original location/callnumber `tr` to remove *its* border, but I only want to do that if there are actually holdings fields there to show.

So it's a bit of a fiddle, but the upshot is that the class only gets added when there are holdings notes to show, and then it's only relevant to bootstrap5 themes.